.neuro/live-s3.yml

```
@@ -0,0 +1,314 @@
+kind: live
+title: demo-oss-dogs
+id: mlops_demo_oss_dogs
+
+defaults:
+  preset: cpu-medium
+  life_span: 1d
+  env:
+    # Those env vars will be assigned to every job in this flow
+    PROJECT_GIT_REPO: git@github.com:neuro-inc/mlops-demo-oss-dogs.git
+    PROJECT_GIT_BRANCH: ${{ git.branch }}
+    BUCKET_NAME: mlops-oss-demo-dogs
```

```suggestion
    BUCKET_NAME: mlops-demo-oss-dogs
```
that was quite a trick to debug :D